pandas/tests/indexing/multiindex/test_setitem.py

```diff
@@ -159,7 +159,7 @@ def test_multiindex_setitem(self):
         )
 
         expected = df_orig.copy()
-        expected.iloc[[0, 2, 3]] *= 2
+        expected.iloc[[0, 1, 3]] *= 2
```

Comment:
what was this returning before?

Reply:
In the past, even with Py36 and above, the results were automatically sort, so there was no problem with this test. But when I solved the automatic sort, I had a problem with this test.

```python
"price": {
                    ("DE", "Coal", "Stock"): 2,
                    ("DE", "Gas", "Stock"): 4,
                    ("DE", "Elec", "Demand"): 1,
                    ("FR", "Gas", "Stock"): 5,
                    ("FR", "Solar", "SupIm"): 0,
                    ("FR", "Wind", "SupIm"): 0,
                }
```
To be specific, data entered in order [Coal, Gas, Elec] will automatically be sort and entered as [Coal, Elec, Gas]. So it seems that the contributor was trying to think and test this, and he used iloc[0, 2, 3] to get direct access to the 'Stock' value position. But as I fixed the unnecessary sort, I had modified it to [0, 1, 3].

But this will cause an error in the Py35 which automatically causes a sort.
So I'm going to fix it now.